src/plus/integrations/providers/models.ts

```diff
@@ -594,65 +594,52 @@ export function getReasonsForUserIssue(issue: ProviderIssue, userLogin: string):
 	return reasons;
 }
 
-export function toSearchedIssue(
-	issue: ProviderIssue,
-	provider: ProviderReference,
-	filterUsed?: IssueFilter,
-	userLogin?: string,
-): SearchedIssue | undefined {
+export function toSearchedIssue(issue: ProviderIssue, provider: ProviderReference): IssueShape | undefined {
```

Comment:
Probably to rename to something like:

```suggestion
export function toIssueShape(issue: ProviderIssue, provider: ProviderReference): IssueShape | undefined {
```

Reply:
done